codec-http2/src/test/java/io/netty/handler/codec/http2/Http2MultiplexCodecTest.java

```diff
@@ -170,6 +170,16 @@ public void inboundDataFrameShouldEmitWindowUpdateFrame() {
         verifyFramesMultiplexedToCorrectChannel(inboundStream, inboundHandler, 2);
     }
 
+    @Test
+    public void unhandledHttp2FramesShouldBePropagated() {
+        ByteBuf content = UnpooledByteBufAllocator.DEFAULT.buffer(8).writeLong(0);
```

Comment:
btw, do you need to free this content?

Reply:
Do you mean for the test, or for the `Http2MultiplexedCodec`?

Reply:
It doesn't look like it's released. It should be. That's why the CI test fails with "Build failure on specific text in build log: Resource leak detected".